finetune_using_clm.py

```diff
@@ -291,6 +297,14 @@ def main(cfg: DictConfig):
     )
 
     # Prepare everything using our accelerator
+    model = model.to(device)
+    for data, target in train_dataloader:
```

Comment:
Gives error: Too many values to unpack (expected 2) for the EleutherAI/gpt-neo-1.3B. Will try with some other models and report further.

Reply:
Doesn't work with distilgpt2 either. Would be great if you can check and update